pkg/sql/logictest/testdata/logic_test/trigram_indexes

```diff
@@ -143,3 +143,17 @@ INSERT INTO pkt VALUES (1, 'abcd'), (2, 'bcde')
 
 statement error primary key column b cannot be present in an inverted index
 ALTER TABLE pkt ALTER PRIMARY KEY USING COLUMNS (b)
+
+# Ensure that it's okay to perform an inverted filter on a table with a trigram
+# inverted index that only has a forward statistic collected on the inverted
+# column.
+
+statement ok
+CREATE TABLE b (a) AS SELECT 'foo';
+ANALYZE b;
+CREATE INVERTED INDEX ON b(a gin_trgm_ops)
```

Comment:
Don't you need to analyze after the index is created? The inverted column doesn't exist until the index is created.

Reply:
The bug was explicitly in this state, where there is an inverted index but a forward histogram on the column that's being inverted indexed. Once you re-analyze, there is no bug. I've added a test case to ensure that both scenarios work okay.